03/app.js

```diff
@@ -1,58 +1,85 @@
 const books = {
-    '978-83-7278-000-3': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Philosopher\'s Stone',
-            pl: 'Harry Potter i Kamień Filozoficzny',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
+  "978-83-7278-000-3": {
+    author: "J.K. Rowling",
+    title: {
+      en: "Harry Potter and the Philosopher's Stone",
+      pl: "Harry Potter i Kamień Filozoficzny",
     },
-    '83-7278-007-2': {
-        author: 'J.K. Rowling',
-        title: {
-            en: 'Harry Potter and the Chamber of Secrets',
-            pl: 'Harry Potter i Komnata Tajemnic ',
-        },
-        translator: {
-            en: null,
-            pl: 'Andrzej Polkowski',
-        }
+    translator: {
+      en: null,
+      pl: "Andrzej Polkowski",
     },
-}
+  },
+  "83-7278-007-2": {
+    author: "J.K. Rowling",
+    title: {
+      en: "Harry Potter and the Chamber of Secrets",
+      pl: "Harry Potter i Komnata Tajemnic ",
+    },
+    translator: {
+      en: null,
+      pl: "Andrzej Polkowski",
+    },
+  },
+};
+
+books.getAuthor = function (isbn) {
+  // w tym przypadku this === books
+
+  if (typeof this[isbn] === "undefined") {
+    // taki ISBN nie istnieje w moim spisie
+    return null;
+  }
+
+  const author = this[isbn]["author"];
+  if (author) {
+    // zwracam informacje o autorze
+    return author;
+  }
+
+  // brak informacji o autorze
+  return false;
+};
+
+books.getTitle = function (isbn, lang) {
+  if (typeof this[isbn] === "undefined") {
+    return null;
+  }
+
+  const title = this[isbn]["title"][lang];
 
-books.getAuthor = function(isbn) {
-    // w tym przypadku this === books
-    
-    if(typeof this[isbn] === 'undefined') {
-        // taki ISBN nie istnieje w moim spisie
-        return null;
-    }
+  if (typeof this[isbn]["title"] === "undefined") {
```

Comment:
Skoro najpierw sprawdzasz właściwość o nazwie `title` to przypisanie do zmiennej `title` powinnaś wykonać poniżej :)